src/requirements/types.ts

```diff
@@ -25,7 +25,7 @@ export type typeOfMigration = 'Modify' | 'Delete' | 'Add';
 
 export type RequirementMigration = {
   entryYear: number /** This migration applies to students with an entryYear equal to or EARLIER this entry year */;
-  type: typeOfMigration /** Modify or Delete Migration? This field must already exist in requirements file */;
+  type: typeOfMigration /** Modify, Delete, or Add Migration? This field must already exist in requirements file */;
```

Comment:
Good catch here with the enum update. Honestly maybe we should just get rid of this since duplication is evil and TypeScript serves the purpose of that part of the comment. But this is a commenting style nitpick that was already here before you, just sharing here for reference for our TPM.